frontend/src/components/candidateDropdown.js

```diff
@@ -26,26 +31,23 @@ const mapDispatchToProps = dispatch => {
   )
 }
 
-type State = {
-  isClearable: boolean,
-  isSearchable: boolean
-}
-
 // this component is used in the interview page for the user to choose candidate
 class CandidateDropdown extends Component<Props, State> {
-  constructor(props) {
-    super(props)
-  }
   state = {
     isClearable: false,
     isSearchable: true,
     selectedOption: null
   }
+
+  constructor(props) {
+    super(props)
+  }
+
```

Comment:
jw, was there a reason this was moved?

Reply:
I thought that initial state declarations should be the first thing in the component class, so that its easier to track the variables. Let me know what you think!

Reply:
no real preference on my end as long as its consistent (which it seems to be), so sounds good!